src/environment/stella_environment.cpp

```diff
@@ -271,15 +271,20 @@ void StellaEnvironment::processScreen() {
     m_phosphor_blend.process(m_screen);
   }
   else {
-    // Copy screen over and we're done! 
-    memcpy(m_screen.getArray(), 
+    // Copy screen over and we're done!
+    memcpy(m_screen.getArray(),
       m_osystem->console().mediaSource().currentFrameBuffer(), m_screen.arraySize());
   }
 }
 
 void StellaEnvironment::processRAM() {
   // Copy RAM over
   for (size_t i = 0; i < m_ram.size(); i++)
-    *m_ram.byte(i) = m_osystem->console().system().peek(i + 0x80); 
+    *m_ram.byte(i) = m_osystem->console().system().peek(i + 0x80);
 }
 
+void StellaEnvironment::processBackRAM() {
```

Comment:
This feels like the wrong interface. Why would you have two separate methods -- one for setting the RAM as read by the ALE, and another one to then copy it back to the emulator? The alternative (set the emulator RAM directly, mirror in ALE) seems much more natural.